demo/app.js

```diff
@@ -961,6 +991,18 @@ const Demo = React.createClass({
         />
         <br/><br/>
         <SearchInput />
+        <DisplayInput
+          hint='Click to Edit'
+          label='Display Input'
+          onBlur={this._handleInputStatusMessage}
+          onFocus={this._handleInputFocus}
+          onMouseOut={this._handleInputHideHint}
+          onMouseOver={this._handleInputShowHint}
```

Comment:
This is the best way I could find to handle a hover state. Is there a better way?